### PR TITLE
[v6r14] Several fixes

### PR DESCRIPTION
**Pull request by Sbalbp**
Changed do_install in SysAdminCLI so that installing a component given a module ( -m flag ) also adds the module information from the configTemplate to the CS entry for the component installed

### CODE REVIEW

**Comment by fstagni** [comment]
Ok.

File: Core/Utilities/InstallTools.py [hunk 1]
```diff
@@ -2481,7 +2481,7 @@ def installDatabase( dbName, monitorFlag = True ):
 
   # now creating the Database
   result = execMySQL( 'CREATE DATABASE `%s`' % dbName )
-  if not result['OK']:
+  if not result['OK'] and not 'database exists' in result[ 'Value' ]:
```
Comment (atsareg) [inline Core/Utilities/InstallTools.py]: If result['OK'] == False there is no guarantee that result has 'Value' key

Comment (Sbalbp) [inline Core/Utilities/InstallTools.py]: It was supposed to be Message, not Value. Thanks for spotting it.

**Comment by fstagni** [comment]
Review OK

**Pull request merged**